src/index.html

```diff
@@ -1,12 +1,464 @@
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__content">
+    <header class="header">
+      <a
+        href="#home"
+        class="header__image"
+      >
+        <img
+           src="images/logo.png"
+           alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#laptopsAndComputers"
+              class="nav__link"
+              data-qa="nav-hover"
+             >
+             Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="container">
+        <div
+        class="container__card card"
+        data-qa="card"
+        >
+          <img
+            src="images/imac.jpeg"
+            alt="imac"
+            class="card__image"
+          >
+
+          <h1 class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h1>
+
+          <p class="card__code">
+              Product code: 195434
+          </p>
+
+          <div class="card__rate">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__reviews">
+              Reviews: 5
+            </p>
+          </div>
+
+          <div class="card__amount">
+            <p class="card__price">
+              Price:
+            </p>
+
+            <h2 class="card__money">
+              $2,199
+            </h2>
+          </div>
+
+          <a
+            href="#buy"
+            class="card__link"
+            data-qa="card-hover"
+          >
+          buy
+          </a>
+        </div>
+
+        <div
+        class="container__card card"
+        >
```

Comment:
Pat attention to formatting your HTML code!

I won't approve this PR till you fix the code formatting in all places